adapters/yieldlab/const.go

```diff
@@ -4,4 +4,5 @@ const adSlotIdSeparator = ","
 const adsizeSeparator = "x"
 const adSourceBanner = "<script src=\"%v\"></script>"
 const adSourceURL = "https://ad.yieldlab.net/d/%v/%v/%v?%v"
+const vastMarkup = "<VAST version=\"2.0\"><Ad id=\"1234567890\"><Wrapper><AdSystem>Yieldlab</AdSystem><VASTAdTagURI><![CDATA[ %v ]]></VASTAdTagURI><Impression></Impression><Creatives></Creatives></Wrapper></Ad></VAST>"
```

Comment:
I was wondering if you could explain why is the Ad id hard coded here? `<Ad id=\"1234567890\">`

I see this in a couple places, so I'll leave the same comments there as well.

Reply:
The ID was hardcoded because the ORTB specs require an ID (otherwise we would simply have omitted it). Our system doesn't generate a unique ID outside of Prebid either though  - when we receive just a VAST-URL, we put a VAST wrapper around it (similar to what we do now for the adm field) that uses the adslot ID as the ad ID. I have now updated the code here accordingly, so our behaviour is identical in both cases.